engine/access/state_stream/backend/handler.go

```diff
@@ -11,29 +11,31 @@ import (
 	"github.com/onflow/flow/protobuf/go/flow/executiondata"
 
 	"github.com/onflow/flow-go/engine/access/state_stream"
+	"github.com/onflow/flow-go/engine/access/subscription"
 	"github.com/onflow/flow-go/engine/common/rpc"
 	"github.com/onflow/flow-go/engine/common/rpc/convert"
 	"github.com/onflow/flow-go/model/flow"
 )
 
 type Handler struct {
+	subscription.StreamingData
+
 	api   state_stream.API
 	chain flow.Chain
 
-	eventFilterConfig state_stream.EventFilterConfig
-
-	maxStreams               int32
-	streamCount              atomic.Int32
+	eventFilterConfig        state_stream.EventFilterConfig
 	defaultHeartbeatInterval uint64
 }
 
 func NewHandler(api state_stream.API, chain flow.Chain, config Config) *Handler {
 	h := &Handler{
+		StreamingData: subscription.StreamingData{
```

Comment:
I think we can benefit from default constructor for `StreamingData`, this code is used at least twice